SightLocator/res/layout-land/text_fragment.xml

```diff
@@ -7,6 +7,20 @@
         android:layout_width="fill_parent"
         android:layout_height="wrap_content"
         android:orientation="vertical" >
+        
+         <TextView
+            android:id="@+id/tilko_vi_lvovi"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginRight="10dp"
+            android:clickable="true"
+            android:freezesText="true"
+            android:text="Tilko_vi_lvovi"
```

Comment:
android:text="Tilko_vi_lvovi":
hardcoded text should be avoided, instead set the text value (from strings.xml) in the code